packages/eslint-plugin-qwik/tests/valid-lexical-scope/invalid-capture-class.tsx

```diff
@@ -5,7 +5,7 @@ export const HelloWorld = component$(() => {
   class Stuff {}
   const stuff = new Stuff();
   useTask$(() => {
-    // eslint-disable-next-line no-console
+    /* eslint no-console: [, { allow: ["log"] }] */
```

Comment:
Is there a reason the disable needs to be replaced?

Reply:
I have upgraded the grade of dependent, `eslint-disable-next-line no-console` will break the unit tests